Command.fs

```diff
@@ -34,9 +34,9 @@ module Types =
         /// <summary>A diverging assertion.</summary>
         | Assume of assumption : BoolExpr<'RV>
         /// <summary>A conditional.</summary>
-        | Branch of conditional : BoolExpr<'RV>
-                  * ifTrue : Microcode<'L, 'RV> list
-                  * ifFalse : Microcode<'L, 'RV> list
+        | Branch of cond : BoolExpr<'RV>
+                  * trueBranch : Microcode<'L, 'RV> list
+                  * falseBranch : Microcode<'L, 'RV> list
```

Comment:
This was just to keep the terminology consistent across other conditional structures.